library/src/main/java/asia/ivity/android/marqueeview/MarqueeView.java

```diff
@@ -213,6 +213,7 @@ private void prepareAnimation() {
         mPaint.setTextSize(mTextField.getTextSize());
         mPaint.setTypeface(mTextField.getTypeface());
         float mTextWidth = mPaint.measureText(mTextField.getText().toString());
+        textViewVirtualWidth = (int) mTextWidth;
```

Comment:
Nit: this is technically the actual width, not a virtual width. :) Probably worth updating to avoid potential future confusion.

Reply:
Yeah @BenHenning this makes sense.